libs/content/workspace/src/space_inspector/show.rs

```diff
@@ -0,0 +1,494 @@
+                    x: current_position + (item.portion * (root_anchor.max.x - root_anchor.min.x)),
+                    y: root_anchor.min.y - ((current_layer - 1) as f32) * self.layer_height,
+                },
+            };
+
+            let current_color = self
+                .colors
+                .iter()
+                .find_map(|element| if element.id == item.id { Some(element.color) } else { None })
+                .unwrap_or(SpaceInspector::get_color(
+                    self,
+                    item.id,
+                    current_layer as usize,
+                    child_number - 1,
+                ));
+
+            // Folder text logic
+            if window.contains(paint_rect.min) {
+                let tab_intel: egui::WidgetText = egui::RichText::new(item.name.clone())
+                    .font(egui::FontId::monospace(0.2 * self.layer_height))
+                    .color({
+                        let hsl_color = colors_transform::Rgb::from(
+                            current_color.r().into(),
+                            current_color.g().into(),
+                            current_color.b().into(),
+                        )
+                        .to_hsl();
+                        let luminance: f32 = if hsl_color.get_lightness() > 50.0 {
+                            (hsl_color.get_lightness() - 50.0) / 100.0
+                        } else {
+                            (hsl_color.get_lightness() + 50.0) / 100.0
+                        };
+                        Color32::from_hex(
+                            &(color_art::color!(
+                                HSL,
+                                hsl_color.get_hue(),
+                                hsl_color.get_saturation() / 100.0,
+                                luminance
+                            ))
+                            .hex(),
+                        )
+                        .unwrap_or(Color32::DEBUG_COLOR)
+                    })
+                    .into();
+
+                let tab_intel_galley = tab_intel.into_galley(
+                    ui,
+                    Some(TextWrapMode::Truncate),
+                    paint_rect.width() - 5.0,
+                    egui::TextStyle::Body,
+                );
+
+                let tab_intel_rect = egui::Align2::LEFT_TOP.anchor_size(
+                    Pos2 { x: paint_rect.left_center().x + 5.0, y: paint_rect.left_center().y },
+                    tab_intel_galley.size(),
+                );
+
+                // painting info
+                painter.clone().rect(
+                    paint_rect,
+                    Rounding::ZERO,
+                    current_color,
+                    Stroke {
+                        width: 0.5,
+                        color: if ui.visuals().dark_mode {
+                            ui.visuals().extreme_bg_color
+                        } else {
+                            Color32::WHITE
+                        },
```

Comment:
```suggestion
                        color: ui.visuals().extreme_bg_color
```
kind of the point of using these fn's is that they already know when you're in dark mode